app/src/organisms/ErrorRecoveryFlows/RecoveryDoorOpen.tsx

```diff
@@ -32,21 +33,71 @@ export function RecoveryDoorOpen({
   recoveryActionMutationUtils,
   runStatus,
   routeUpdateActions,
+  recoveryMap,
+  recoveryCommands,
 }: RecoveryContentProps): JSX.Element {
   const {
     resumeRecovery,
     isResumeRecoveryLoading,
   } = recoveryActionMutationUtils
-  const { stashedMap, proceedToRouteAndStep } = routeUpdateActions
+  const {
+    stashedMap,
+    proceedToRouteAndStep,
+    handleMotionRouting,
+  } = routeUpdateActions
+  console.log('recoveryMap: ', recoveryMap)
   const { t } = useTranslation('error_recovery')
 
+  const handleHomeAllAndRoute = (
+    route: RecoveryRoute,
+    step?: RouteStep
+  ): void => {
+    void handleMotionRouting(true, RECOVERY_MAP.ROBOT_IN_MOTION.ROUTE)
```

Comment:
```suggestion
    void handleMotionRouting(true)
```
If it's the default case, let's not specify the route for clarity.